From: SeongJae Park <sj@kernel.org>
Date: Mon, 1 Jul 2024 15:52:17 -0700
Subject: [PATCH] MAINTAINERS/MEMORY MANAGEMENT: add document files for mm as
 managed files

Add 'Documentation/admin-guide/mm/' and 'Documentation/mm/' to 'F:'
entries for 'MEMORY MANAGEMENT' section, so that get_maintainer.pl
suggests appropriate maintainers for the files.

Signed-off-by: SeongJae Park <sj@kernel.org>
---
 MAINTAINERS | 2 ++
 1 file changed, 2 insertions(+)

diff --git a/MAINTAINERS b/MAINTAINERS
index 79a6e8cd4f0c..1074637b3e65 100644
--- a/MAINTAINERS
+++ b/MAINTAINERS
@@ -14794,6 +14794,8 @@ S:	Maintained
 W:	http://www.linux-mm.org
 T:	git git://git.kernel.org/pub/scm/linux/kernel/git/akpm/mm
 T:	quilt git://git.kernel.org/pub/scm/linux/kernel/git/akpm/25-new
+F:	Documentation/admin-guide/mm/
+F:	Documentation/mm/
 F:	include/linux/gfp.h
 F:	include/linux/gfp_types.h
 F:	include/linux/memfd.h
-- 
2.39.5

